### PR TITLE
[TS] LPS-152798

### PR DESCRIPTION
**Pull request by jcampoy**
Related Issue: https://issues.liferay.com/browse/LPS-152798

https://issues.liferay.com/browse/LPS-152798

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @jcampoy, can you please check if these failures are related to your changes? Thanks

**Comment by jcampoy** [comment]
the unique failure is not related with my commit

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java [hunk 1]
```diff
@@ -373,10 +373,18 @@ private DDMDataProviderResponse _getData(
...
+		String response = null;
 
-		jsonWebServiceClient.destroy();
+		try {
+			response = jsonWebServiceClient.doGet(
+				absoluteURL, _getParametersArray(allParameters));
+		}
+		catch (JSONWebServiceException jsonWebServiceException) {
+			throw jsonWebServiceException;
+		}
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java:384]: Hi @jcampoy, do we really need this `catch`?  Wouldn't `try/finally` be enough?

Comment (jcampoy) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java:384]: we have to throw the exception, see the caller that is catching and handling them...

Comment (jcampoy) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java:384]: if the try-finally throws up the exception, in that case I am right with the change :D

Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java:384]: The `try` block will throw the exception anyway, but the `catch` would be necessary in this case if we need to handle the exception some other way

Comment (jcampoy) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java:384]: @carolmariaabb I am going to resend the PR with your suggestions! thanks!

Comment (jcampoy) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java:384]: @carolmariaabb the changes have been done

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @jcampoy, can you please check if these failures are related to your changes? Thanks

**Pull request closed without merging**